package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.0.0",
   "description": "electron-react-redux-boilerplate",
   "main": "init.js",
-  "author": "Jordan Schroter",
+  "author": {
+    "name": "Jordan Schroter",
+    "email": "email@author.com"
```

Comment:
@jschr add your email if you wish :) This is needed for .deb package builds for some reason.